tests/utils.py

```diff
@@ -33,15 +33,15 @@
 """
 
 # Python imports.
-import os
 import numpy as np
+import os
 import shutil
 import subprocess
-import warnings
-import xml.etree.ElementTree as ET
 import vtk
 from vtk.util import numpy_support as vtk_numpy
 from vtk_utils.compare_grids import compare_grids
+import warnings
+import xml.etree.ElementTree as ET
```

Comment:
I think these changes are not needed, correct? (are you using a tool that orders the imports?)

Reply:
No they are not needed. removed them as well.